with-dev-client/android/app/src/main/java/com/helloworld/MainApplication.java

```diff
@@ -81,14 +87,14 @@ public ReactNativeHost getReactNativeHost() {
   @Override
   public void onCreate() {
     super.onCreate();
+    DevLauncherController.initialize(this, getReactNativeHost());
     SoLoader.init(this, /* native exopackage */ false);
 
     if (!BuildConfig.DEBUG) {
       UpdatesController.initialize(this);
     }
 
     initializeFlipper(this, getReactNativeHost().getReactInstanceManager());
```

Comment:
```suggestion
    // initializeFlipper(this, getReactNativeHost().getReactInstanceManager());
```
We should make flipper opt-in rather then opt-out

Reply:
On iOS flipping Flipper on/off is a bit more flexible, because we check if `FB_SONARKIT_ENABLED` is defined and Flipper only gets initialized if the dependency is included. If we only comment out this line, I believe it would still be compiled?

It seems that Flipper is less problematic on Android than iOS. However, if we want to do this for Android, the right way to do it would be to make the change in [expo-template-bare-minimum](https://github.com/expo/expo/tree/master/templates/expo-template-bare-minimum) and then re-run `expo prebuild --clean` in this project to sync it here.